src/Components/Buttons/OulinedButton.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { colors } from '../../theme/colors'
```

Comment:
In some places imports are made with single quote and in some places with double quotes, please stick with one and use that in entire code.